docker-build.sh

```diff
@@ -140,13 +140,13 @@ EOF
     yes | apt-get install -y -o APT::Immediate-Configure=0 gcc-${GCC_VER}-source libstdc++6-armhf-cross binutils-arm-linux-gnueabihf bison flex libtool gdb sharutils netbase libmpc-dev libmpfr-dev libgmp-dev systemtap-sdt-dev autogen expect chrpath zlib1g-dev zip libc6-dev:armhf linux-libc-dev:armhf libgcc1:armhf libcurl4-openssl-dev:armhf libfontconfig1-dev:armhf libfreetype6-dev:armhf liblttng-ust0:armhf libstdc++6:armhf
     popd
 
-    # Fetch RasPi headers to build MMAL support
+    # Fetch RasPi headers to build MMAL and OMX-RPI support
     pushd ${SOURCE_DIR}
-    git clone --depth=1 https://github.com/raspberrypi/firmware mmalheaders
-    git clone --depth=1 https://github.com/raspberrypi/userland piuserland
-    cp -a mmalheaders/opt/vc/include/* /usr/include/
-    cp -a mmalheaders/opt/vc/lib/* /usr/lib/
-    cp -a piuserland/interface/* /usr/include/
+    svn checkout https://github.com/raspberrypi/firmware/trunk/opt/vc/include rpi/include
```

Comment:
Why switch to SVN? That just means one more thing to install.

Reply:
Since raspberrypi headers and libs only require 6MB, and the clone repository requires more than 120MB. The same is true for the AMD AMF headers, which requires 100MB in its entirety. This change significantly reduces the long pauses in these preparations before compilation.

Reply:
Couldn't we just `curl`/`wget` the individual files we need instead? Either way there's more downloading involved (either in apt, or in the repo).

Reply:
What I list below is only a small part of the MMAL header file. Using `wget`/`curl` to download files in the git directory and subdirectories one by one may not be ideal. Or is there a better way than SVN that I don't know?

```
├── IL
│   ├── OMX_Audio.h
│   ├── OMX_Broadcom.h
│   ├── OMX_Component.h
│   ├── OMX_Core.h
│   ├── OMX_ILCS.h
│   ├── OMX_Image.h
│   ├── OMX_Index.h
│   ├── OMX_IVCommon.h
│   ├── OMX_Other.h
│   ├── OMX_Types.h
│   └── OMX_Video.h
├── interface
│   ├── debug_sym
│   │   └── debug_sym.h
│   ├── mmal
│   │   ├── core
│   │   │   ├── mmal_buffer_private.h
│   │   │   ├── mmal_clock_private.h
│   │   │   ├── mmal_component_private.h
│   │   │   ├── mmal_core_private.h
│   │   │   ├── mmal_events_private.h
│   │   │   └── mmal_port_private.h
│   │   ├── mmal_buffer.h
│   │   ├── mmal_clock.h
│   │   ├── mmal_common.h
│   │   ├── mmal_component.h
│   │   ├── mmal_encodings.h
│   │   ├── mmal_events.h
│   │   ├── mmal_format.h
│   │   ├── mmal.h
│   │   ├── mmal_logging.h
│   │   ├── mmal_parameters_audio.h
│   │   ├── mmal_parameters_camera.h
│   │   ├── mmal_parameters_clock.h
│   │   ├── mmal_parameters_common.h
│   │   ├── mmal_parameters.h
│   │   ├── mmal_parameters_video.h
│   │   ├── mmal_pool.h
│   │   ├── mmal_port.h
│   │   ├── mmal_queue.h
│   │   ├── mmal_types.h
│   │   ├── util
│   │   │   ├── mmal_component_wrapper.h
│   │   │   ├── mmal_connection.h
│   │   │   ├── mmal_default_components.h
│   │   │   ├── mmal_graph.h
│   │   │   ├── mmal_il.h
│   │   │   ├── mmal_list.h
│   │   │   ├── mmal_param_convert.h
│   │   │   ├── mmal_util.h
│   │   │   ├── mmal_util_params.h
│   │   │   └── mmal_util_rational.h
│   │   └── vc
│   │       ├── mmal_vc_api_drm.h
│   │       ├── mmal_vc_api.h
│   │       ├── mmal_vc_client_priv.h
│   │       ├── mmal_vc_msgnames.h
│   │       ├── mmal_vc_msgs.h
│   │       ├── mmal_vc_opaque_alloc.h
```

Reply:
Yea that looks like a pain, it's enough files that it's too much to "wget -r" cleanly, so I'm OK with it.